lib/units/unit.cpp

```diff
@@ -184,4 +184,11 @@ Unit atan2(const Unit &y, const Unit &x) {
       " b " + y.name() + ".");
 }
 
+Unit addUserDefinedUnit(std::string name, Unit u) {
+  llnl::units::addUserDefinedUnit(name, u.underlying());
+  return Unit(llnl::units::unit_from_string(name));
+}
+
+void clearUserDefinedUnits() { return llnl::units::clearUserDefinedUnits(); }
```

Comment:
We will probably want a more "pythonic" interface. I am thinking, e.g., about something like a `dict`-like than you can insert into, del from, etc. It would also allow for listing all user-defined units. Would probably be best to implement that purely in Python (and keep the functions here as internals, to access the C++ component).

I suggest we have a chat next week when we are back to the office?